data/mods/Xedra_Evolved/effects/effects.json

```diff
@@ -1054,61 +1052,31 @@
       }
     ]
   },
-  {
-    "type": "effect_type",
-    "id": "spell_karma_arms_eff",
-    "name": [ "" ],
-    "desc": [ "" ],
-    "apply_message": "Ethereal, transparent arms grow from your shoulders, and <color_cyan>glow</color>.",
-    "remove_message": "The cyan arms grow dim, and finally disappear.",
-    "rating": "good",
-    "enchantments": [ { "mutations": [ "karma_arms" ] } ]
-  },
-  {
-    "type": "mutation",
-    "id": "stalker_eyes",
-    "name": { "str": "Stalker's Vision" },
-    "points": 1,
-    "visibility": 1,
-    "description": "Your own eyes transform into the orbits of an unknown creature.  You see much better, both in the light and in the dark.",
-    "//": "not actual replacement yet, because i'm afraid of how wearing stuff may interact with it",
-    "valid": false,
-    "starting_trait": false,
-    "purifiable": false,
-    "enchantments": [ { "condition": "ALWAYS", "modified_bodyparts": [ { "gain": "stalker_eyes" } ] } ]
-  },
   {
     "type": "effect_type",
     "id": "stalker_eyes_eff",
-    "name": [ "" ],
-    "desc": [ "" ],
+    "name": [ "Stalker's Vision" ],
+    "desc": [
+      "Your own eyes transform into the orbits of an unknown creature.  You see much better, both in the light and in the dark"
+    ],
     "apply_message": "Your vision gets dark for a moment, and in a second, a whole new world is revealed under your eyes.",
     "remove_message": "A great new world under your eyes disappears.",
     "blocks_effects": [ "glare" ],
     "rating": "good",
-    "enchantments": [ { "mutations": [ "stalker_eyes" ] } ]
-  },
-  {
-    "type": "mutation",
-    "id": "devil_tail",
-    "name": { "str": "Devil's Tail" },
-    "points": 1,
-    "visibility": 6,
-    "description": "Not actually devil's, this long tail with triangle point in the end can stretch for a lot, and could easily kill someone nearby.",
-    "valid": false,
-    "starting_trait": false,
-    "purifiable": false,
-    "enchantments": [ { "condition": "ALWAYS", "modified_bodyparts": [ { "gain": "devil_tail" } ] } ]
+    "//": "not actual replacement yet, because i'm afraid of how wearing stuff may interact with it",
+    "enchantments": [ { "condition": "ALWAYS", "modified_bodyparts": [ { "gain": "stalker_eyes" } ] } ]
   },
   {
     "type": "effect_type",
     "id": "devil_tail_eff",
-    "name": [ "" ],
-    "desc": [ "" ],
+    "name": [ "Devil's Tail" ],
+    "desc": [
+      "Not actually devil's, this long tail with triangle point in the end can stretch for a lot, and could easily kill someone nearby"
```

Comment:
"While not actually a devil's, this long, triangular pointed tail can stretch very far, allowing it to easily kill someone nearby."